src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -133,6 +138,13 @@ export const MyVideosView = () => {
   // Workaround for removing drafts from video sheet tabs via snackbar
   // Snackbar will probably need a refactor to handle actions that change state
   useEffect(() => {
+    if (videoTabs.length === 0) {
+      closeSnackbar(OPEN_TAB_SNACKBAR)
+      return
+    }
+    updateSnackbar(OPEN_TAB_SNACKBAR, {
+      title: `${videoTabs.length > 1 ? `${videoTabs.length} videos` : 'Video'} opened in a new tab`,
```

Comment:
That won't produce the correct output though right? You can have 5 tabs already opened, add only 1, but you will get snackbar that you added 6